plugins/WiseQuote/plugin.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "WiseQuote",
+  "version": "1.0.0",
+  "author": "Bartha",
```

Comment:
this does not seem to be the true author

Reply:
I've added the first initial as per the true author's request.